### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by KatyDi**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -197,38 +197,39 @@ <h1 class="name">Start Bootstrap</h1>
...
+                                <label for="name">Imię</label>
+                                <input type="text" class="form-control" placeholder="Imię" id="name" required data-validation-required-message="Wpisz swoje imię.">
                                 <p class="help-block text-danger"></p>
                             </div>
                         </div>
                         <div class="row control-group">
                             <div class="form-group col-xs-12 floating-label-form-group controls">
-                                <label for="email">Email Address</label>
-                                <input type="email" class="form-control" placeholder="Email Address" id="email" required data-validation-required-message="Please enter your email address.">
+                                <label for="email">Adres email</label>
```
Comment (w1stler) [inline index.html]: Email or e-mail? Please be consitent in naming.

File: index.html [hunk 2]
```diff
@@ -197,38 +197,39 @@ <h1 class="name">Start Bootstrap</h1>
...
+                                <input type="email" class="form-control" placeholder="Adres email" id="email" required data-validation-required-message="Wpisz swój adres email.">
```
Comment (w1stler) [inline index.html]: ditto

File: mail/contact_me.php [hunk 3]
```diff
@@ -16,11 +16,11 @@
...
-$to = 'yourname@yourdomain.com'; // Add your email address inbetween the '' replacing yourname@yourdomain.com - This is where the form will send a message to.
-$email_subject = "Website Contact Form:  $name";
-$email_body = "You have received a new message from your website contact form.\n\n"."Here are the details:\n\nName: $name\n\nEmail: $email_address\n\nPhone: $phone\n\nMessage:\n$message";
-$headers = "From: noreply@yourdomain.com\n"; // This is the email address the generated message will be from. We recommend using something like noreply@yourdomain.com.
-$headers .= "Reply-To: $email_address";	
+$to = 'hello@codeforpoznan.pl'; // Add your email address inbetween the '' replacing hello@codeforpoznan.pl - This is where the form will send a message to.
+$email_subject = "Formularz kontaktowy Code For Poznan:  $name";
+$email_body = "Otrzymałeś nową wiadomość z formularza kontaktowego codeforpoznan.pl.\n\n"."Oto szczegóły:\n\nImię: $name\n\nEmail: $email_address\n\nTelefon: $phone\n\nMessage:\n$message";
+$headers = "From: noreply@codeforpoznan.pl\n"; // To jest adres e-mail, z którego została wygenerowana wiadomość. W celu kontaktu zalecamy użyć hello@codeforpoznan.pl";
```
Comment (w1stler) [inline mail/contact_me.php]: We don't have to translate comments. ;-)

File: index.html (hunk 1 above)
Comment (KatyDi) [reply index.html]: I think I'm consitent. I didn't use 'email' anywhere.

File: mail/contact_me.php (hunk 3 above)
Comment (KatyDi) [reply mail/contact_me.php]: You're right ;)

File: index.html (hunk 1 above)
Comment (w1stler) [reply index.html]: Hmm, my bad then, sorry. ;-)

**Pull request merged**